lib/SPIRV/libSPIRV/SPIRVDecorate.h

```diff
@@ -83,6 +83,10 @@ class SPIRVDecorateGeneric : public SPIRVAnnotationGeneric {
       BuiltIn BI = static_cast<BuiltIn>(Literals.back());
       return getCapability(BI);
     }
+    case DecorationUniform:
```

Comment:
We don't have `DecorationUniformId` enlisted in `lib/SPIRV/libSPIRV/SPIRVEnum.h` yet it seem to be known to the translator (at least in `lib/SPIRV/libSPIRV/SPIRVNameMapEnum.h`). Shouldn't we add the appropriate check for it as well?